README.md

```diff
@@ -48,9 +48,30 @@ To do this, set:
 
 For example:
 
-    vault write secret/foo value=bar other=baz
+    vault write secret/foo value=bar
+
+The hiera lookup for `foo` will return just "bar" as a string, but only if it is the only field.
+If `foo` contains more fields, a Hash will be returned, just like with the default behaviour.
+
+### json parsing of single values - optional
+Only applicable when :default_field is used.
+To use json parsing, set, for example:
```

Comment:
You need an empty new line before the indented code snippet or it doesn't render correctly.